### DISCUSSION TITLE
o3de and embergen

### DISCUSSION THREAD

**Discussion by jeffpenn82** [o3de/o3de#16694] [Help Wanted]
Has anyone used embergen and o3de? I am trying to see if I can add a cool explosion from embergen into o3de.

**Comment by AMZN-alexpete** [comment]
I haven't used Embergen with O3DE yet but if can [export FlipBooks](https://docs.jangafx.com/embergen/pages/references/node_list.html#exporting) and you can use [PopcornFX ](https://www.popcornfx.com/plugin-o3de/)to [render/control those](https://youtu.be/WCLiV2_eKxo).
